### PR TITLE
add support for new API removeConnectionFromAllGroups

### PR DESCRIPTION
**Pull request by xingsy97**
### Summary of the changes (Less than 80 chars)
 - add support for new API `removeConnectionFromAllGroups`
 - add corresponding unit test (both Transient mode & Persistent mode)

### CODE REVIEW

File: src/Microsoft.Azure.SignalR.Common/Logging/MessageLog.cs [hunk 1]
```diff
@@ -118,6 +119,12 @@ internal static class MessageLog
                 new EventId(91, "StartToRemoveUserFromAllGroups"),
                 StartToRemoveUserFromAllGroupsTemplate);
 
+        private static readonly Action<ILogger, ulong?, string, Exception> _startToRemoveConnectionFromAllGroups =
+            LoggerMessage.Define<ulong?, string>(
+                LogLevel.Information,
+                new EventId(91, "StartToRemoveConnectionFromAllGroups"),
```
Comment (vwxyzh) [inline src/Microsoft.Azure.SignalR.Common/Logging/MessageLog.cs:125]: Duplicated: `91`

File: src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs [hunk 2]
```diff
@@ -161,6 +161,22 @@ public override Task<IActionResult> RemoveConnectionFromGroup([RegularExpression
             return Task.FromResult(NotFound() as IActionResult);
         }
 
+        public override Task<IActionResult> RemoveConnectionFromAllGroups([RegularExpression(HubPattern)] string hub, string connection)
+        {
+            if (!ModelState.IsValid)
+            {
+                return Task.FromResult(BadRequest() as IActionResult);
```
Comment (vwxyzh) [inline src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs:168]: why not `return Task.FromResult<IActionResult>(BadRequest());`

Comment (xingsy97) [reply src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs:168]: fixed

File: src/Microsoft.Azure.SignalR.Common/Logging/MessageLog.cs (hunk 1 above)
Comment (xingsy97) [reply src/Microsoft.Azure.SignalR.Common/Logging/MessageLog.cs]: fixed

File: src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs (hunk 2 above)
Comment (xingsy97) [reply src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs:168]: Simply follow the code style of other methods in the same file(`SignalRServiceEmulatorWebApi.cs`).  For example: [L84](https://github.com/Azure/azure-signalr/blob/417fa6407b70b78c33b18ab3a0f21cbf0dcc84f9/src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs#L84) , [L263](https://github.com/Azure/azure-signalr/blob/417fa6407b70b78c33b18ab3a0f21cbf0dcc84f9/src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs#L263)

File: src/Microsoft.Azure.SignalR.Management/RestApiProvider.cs [hunk 3]
```diff
@@ -105,5 +110,14 @@ private async Task<RestApiEndpoint> GenerateRestApiEndpointAsync(string appName,
             var token = await _restApiAccessTokenGenerator.Generate($"{audiencePrefixWithHub}{pathAfterHub}", lifetime);
             return new RestApiEndpoint($"{requestPrefixWithHub}{pathAfterHub}", token) { Query = queries };
         }
+
+        private async Task<RestApiEndpoint> GenerateRestApiEndpointWithVersionAsync(string appName, string hubName, string pathAfterHub, string apiVersion, TimeSpan? lifetime = null, IDictionary<string, StringValues> queries = null)
```
[the remark this answers is not in the document]
Comment (xingsy97) [reply src/Microsoft.Azure.SignalR.Management/RestApiProvider.cs]: PR #1646 (generate API endpoints for V20220601) and PR #1647 (add swagger for V20220601) were merged now.

File: src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs (hunk 2 above)
Comment (vicancy) [reply src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs:168]: Let's have another PR to batch update them

**Review by vicancy — APPROVED** [review]

**Pull request merged**